pkgs/by-name/ed/edk2/package.nix

```diff
@@ -48,29 +46,27 @@ edk2 = stdenv.mkDerivation rec {
     })
   ];
 
-  srcWithVendoring = fetchFromGitHub {
+  src = fetchFromGitHub {
     owner = "tianocore";
     repo = "edk2";
     rev = "edk2-stable${edk2.version}";
     fetchSubmodules = true;
-    hash = "sha256-Nurm6QNKCyV6wvbj0ELdYAL7mbZ0yg/tTwnEJ+N18ng=";
+    hash = "sha256-7vNodHocwqQiO0ZXtqo8lEOFyt8JkFHcAathEhrKWE0=";
```

Comment:
Please verify if hash here should be:
`sha256-4JDeA4QBa/FYqgHi5l6FT9EdcTtFvIbKzO+ywT1BsC8=`

Reply:
Would it make sense to add `edk2` package to `openssl` passthru.tests, such that, whenever someone is bumping `openssl` is also going to be hinted that `edk2` hash should be bumped too?